src/events.js

```diff
@@ -2294,6 +2285,8 @@ const proccedToSpecimenPage = async (accessionID1, accessionID3, selectedVisit,
             const yesBtn = document.getElementById('addCollection');
             yesBtn.addEventListener("click", async e => {
                 formData.collectionId = bloodAccessionId?.data?.[820476880];
+                formData['331584571'] =  selectedVisit;
```

Comment:
was this CID not getting set properly in this scenario?

Reply:
True. It is not set correctly. This is  1 of the 2 lines needed for the notification fix.